android/brave_java_sources.gni

```diff
@@ -5,6 +5,8 @@
 
 import("//brave/android/features/tab_ui/brave_tab_management_java_sources.gni")
 import("//brave/android/feed/brave_feed_java_sources.gni")
+import(
+    "//brave/android/java/org/chromium/chrome/browser/preferences/brave_java_sources.gni")
```

Comment:
Also since we patch `chrome/browser/preferences/BUILD.gn` shouldn't out file be `//brave/browser/preferences/brave_preferences_java_sources.gni`

Reply:
Chromium class is at 
`src/chrome/browser/preferences/android/java/src/org/chromium/chrome/browser/preferences/SharedPreferencesManager.java`

Brave's class is at
`src/brave/android/java/org/chromium/chrome/browser/preferences/BravePreferenceKeyChecker.java`

So I put `brave_java_sources.gni` near the `BravePreferenceKeyChecker.java`.

But yes, Chromium's `BUILD.gn` is at `src/chrome/browser/preferences/BUILD.gn`.
And our naming convention is to repeat the Chromium's scheme.

This means you are right, I will have
`//brave/browser/preferences/brave_preferences_java_sources.gni`
still with `brave_browser_preferences_java_sources = [ "//brave/android/java/org/chromium/chrome/browser/preferences/BravePreferenceKeyChecker.java" ]`

Doing the change.

Reply:
Fixed at https://github.com/brave/brave-core/compare/fbc99007298ba27e53f594f56c6bd8374b904a9b..de51521bf4b2d667fedb65c20c07907872ab08ba